test/fixtures/sanitation/response-body-attribute.js

```diff
@@ -1,40 +1,28 @@
 const hooks = require('hooks');
 
-hooks.after('Resource > Update Resource', (transaction, done) => {
-  // Sanitation of the attribute in body
-  let body;
+const unfold = (jsonString, transform) => JSON.stringify(transform(JSON.parse(jsonString)));
 
-  body = JSON.parse(transaction.test.actual.body);
-  delete body.token;
-  transaction.test.actual.body = JSON.stringify(body);
+hooks.after('Resource > Update Resource', (transaction, done) => {
+  const deleteToken = (obj) => {
+    delete obj.token;
+    return obj;
+  };
 
-  body = JSON.parse(transaction.test.expected.body);
-  delete body.token;
-  transaction.test.expected.body = JSON.stringify(body);
+  // Removes sensitive data from the Dredd transaction
+  const nextBody = unfold(transaction.test.actual.body, deleteToken);
```

Comment:
I'd rename this to `sanitizedBody`

Reply:
Oh, I would remove this at all. The next line illustrates how this property can be re-assigned.